src/pkg/backup/details/details.go

```diff
@@ -1044,3 +1044,8 @@ func updateFolderWithinDrive(
 
 	return nil
 }
+
+// ExtensionInfo describes extension data associated with an item
+type ExtensionInfo struct {
+	Data map[string]any `json:"data,omitempty"`
```

Comment:
Suggestion: check out the `xsync` package I'm using for synchronous mapping in [this pr](https://github.com/alcionai/corso/pull/3722/files#diff-f0179d0b53f0e6169d90c6b0904c7988aa126b0ed82135bf2c0950473e9679beR4) ([github](https://github.com/puzpuzpuz/xsync)).  It might save us some concurrency handling concerns.

Reply:
Missed this one. 100%. Thanks for the suggestion!